Packs/PANWComprehensiveInvestigation/ReleaseNotes/1_3_11.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Palo Alto Networks - Hunting And Threat Detection
+Fixed an issue with pointing the query to a global context path (File) instead of the playbook inputs.
```

Comment:
Fixed an issue where the query pointed to the global context path instead of the playbook inputs.